org.kuchelmeister.ToolboxTuner.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id" : "org.kuchelmeister.ToolboxTuner.Devel",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm15"
+    ],
+    "command" : "toolbox-tuner",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
why is this necessary.

Reply:
The application needs to talk to certain host applications via the commandline. These are _toolbox_ and _podman_.  Also it is used to open the terminal. The commands used for achieving this are usually something like: 
* `flatpak-spawn --host gnome-terminal --version`
* `flatpak-spawn --host podman stop <hash>`
* `flatpak-spawn --host toolbox list`

I'd appreciate suggestions on how to avoid using  "--talk-name=org.freedesktop.Flatpak"

Reply:
You need to ask an exception to this https://docs.flathub.org/docs/for-app-authors/linter#exceptions

The error code is https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-spawn-access